cmd/snaptel/config.go

```diff
@@ -40,6 +41,7 @@ type restAPIConfig struct {
 }
 
 func (c *config) loadConfig(path string) error {
+	log.Warning("The snaptel configuration file will be deprecated. Find more information here: https://intelsdi.atlassian.net/browse/SDI-2580 ")
```

Comment:
Let's move this to a github issue link since not everyone have access to Alassian. Otherwise this PR LGTM.